sdk/client-core/core/src/main/java/io/clientcore/core/implementation/util/Slf4jLoggerShim.java

```diff
@@ -214,7 +215,10 @@ private static Object createLogger(String className) {
     }
 
     private static void writeSlf4jDisabledError(ClientLogger.LogLevel level, String message, Throwable throwable) {
-        DEFAULT_LOGGER.log(level, String.format("[DefaultLogger]: %s. SLF4J logging will be disabled.", message),
-            throwable);
+        if (!slf4jErrorLogged) {
+            slf4jErrorLogged = true;
```

Comment:
no synchronization needed - it's not the end of the world if we log it twice per process lifetime because of a race-condition.